src/main/scala/centaur/test/standard/StandardTestFormat.scala

```diff
@@ -0,0 +1,42 @@
+package centaur.test.standard
+
+import cats.data.Validated.{Valid, _}
+import centaur.test.{ErrorOr, TestFormulas}
+import com.typesafe.config.Config
+import lenthall.config.ScalaConfig._
+
+sealed abstract class StandardTestFormat {
+  def testSpecString = this match {
+    case WorkflowSuccessTest => "successfully run"
+    case WorkflowFailureTest => "fail during execution"
+    case SubmissionFailureTest => "fail to submit"
+  }
+
+  def testFunction = this match {
+    case WorkflowSuccessTest => TestFormulas.runSuccessfulWorkflow _
+    case WorkflowFailureTest => TestFormulas.runFailingWorkflow _
+    case centaur.test.standard.SubmissionFailureTest => TestFormulas.runSubmissionFailureWorkflow _
```

Comment:
seems like these things should be defined on the instances rather than exploded out and matched on